README.md

```diff
@@ -75,6 +95,7 @@ documentation.
   * **reboot_required**      -  Returns a hash of 'name' => 'reason' entries
   * **runlevel**             -  Return the current system runlevel
   * **shmall**               -  Return the value of shmall from sysctl
+  * **simp_puppet_settings** -  Returns a hash of all Puppet settings on a node
```

Comment:
The fact is called `puppet_settings`, even though the file is called `simp_puppet_settings`. Stdlib has a few facts in a file called `puppet_settings`, neither of which are named `puppet_settings`.